data/json/recipes/recipe_deconstruction.json

```diff
@@ -2016,10 +2016,18 @@
   {
     "result": "cable",
     "type": "uncraft",
-    "time": "2 m",
+    "time": "8 s",
     "//": "This may need to be modified, a plastic sheath would give plastic chunks, but there already is a plastic sheathed cable",
     "activity_level": "LIGHT_EXERCISE",
     "qualities": [ { "id": "CUT", "level": 2 } ],
+    "components": [ [ [ "copper_wire", 1 ] ] ]
+  },
+  {
+    "result": "copper_wire",
+    "type": "uncraft",
+    "time": "1 s",
+    "//": "I made this a near instant conversion.  Raw copper wire is essentially just copper and imo should be freely converted without a huge time investment or tool requirement, they are effectively identical items.",
+    "activity_level": "LIGHT_EXERCISE",
```

Comment:
Smash raw copper wire into copper fragment, with your bare hands, in one second, that's not something realistic.

Reply:
You're not smashing it.  It's not being transformed in any way.  It's just recognizing that a loose raw copper wire is a fragment of copper.  We do this with weak alcohol as an example.

Reply:
However it does look like this would be creating a half a gram of copper out of thin air each round.

Reply:
> You're not smashing it. It's not being transformed in any way. It's just recognizing that a loose raw copper wire is a fragment of copper. We do this with weak alcohol as an example.

I doubt with this, considering that copper is being used in recipes where raw copper wire may not be an appropriate component.

https://cdda-guide.nornagon.net/item/copper

Reply:
Yeah I'll be real, as I mentioned in the issue I'm not doing all of the work required to set this up what has been proposed.  This PR can be closed by someone in authority if that is desired.